Cargo.toml

```diff
@@ -9,3 +9,7 @@ exclude = [
   "wasm/no_std",
 ]
 
+[workspace.dependencies]
+givre = { path = "./givre" }
```

Comment:
You're still importing it by path in tests/Cargo.toml; I assume you add it to workspace deps to be able to do `workspace = true`?

Reply:
Yep that was the purpose, I fixed it